persistence/postgres-persistence/src/main/java/com/netflix/conductor/postgres/config/PostgresConfiguration.java

```diff
@@ -52,6 +52,7 @@ public PostgresConfiguration(DataSource dataSource, PostgresProperties propertie
         this.properties = properties;
     }
 
+    @ConditionalOnProperty(name = "spring.flyway.enabled", havingValue = "true")
     @Bean(initMethod = "migrate")
     @PostConstruct
     public Flyway flywayForPrimaryDb() {
```

Comment:
@v1r3n We need to keep this custom Flyway definition, shouldn't it use Spring's?